components/forms/PasswordReset/index.jsx

```diff
@@ -0,0 +1,31 @@
+import Input from '@/components/Input';
+import Button from '@/components/Button';
+
+export const passwordResetFormTestIds = {
+  form: 'password-reset__form',
+  emailLabel: 'password-reset__email',
+  emailField: 'password-reset__input-email',
+  passwordResetButton: 'password-reset__button',
```

Comment:
How about this?

```suggestion
  form: 'password-reset-form',
  emailLabel: 'password-reset-form__email',
  emailField: 'password-reset-form__input-email',
  passwordResetButton: 'password-reset-form__button',
```

Reply:
done in d129258d877c1e23ba31313cc0390ad9627eb192